gradle/apps/hello-soy-portlet/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"dependencies": {
+		"metal-component": "^2.4.5",
+		"metal-soy": "^2.4.5"
+	},
+	"devDependencies": {
+		"liferay-module-config-generator": "^1.2.1",
+		"metal-cli": "^4.0.1"
+	},
+	"name": "hello-soy-portlet",
```

Comment:
@gamerson, I'm not sure how this works, but there's no `script` block in here. I have no idea how it builds

Reply:
And because of this, I couldn't figure out how to get the maven sample to build correctly.